scripts/iob_wire.py

```diff
@@ -0,0 +1,9 @@
+class iob_wire:
+    """Class for IOB wire"""
+
+    name = ""
+    bit_width = 1
+
+    def __init__(self, name, bit_width):
```

Comment:
```suggestion
    def __init__(self, name, bit_width=1):
```
The `name` and `bit_width` attributes declared outside the `__init__` method are `class` attributes in Python, and therefore share their value with all instances of this class.

The `self.name` and `self.bit_width` attributes declared inside the `__init__` method are `instance` attributes, and therefore are unique to each instance.

Since each `iob_wire` instance will have its own name and bit_width, I don't see the need for the `class` attributes.